test/unit/helpers/bpmn-model-utils.ts

```diff
@@ -88,36 +88,32 @@ const newGroup = (parent: string, id: string, name: string): Shape => new Shape(
 const newSubProcess = (parent: string, id: string, name: string, isExpanded: boolean): Shape =>
   new Shape(buildShapeId(id), new ShapeBpmnSubProcess(id, name, ShapeBpmnSubProcessKind.EMBEDDED, parent, isExpanded ? undefined : [ShapeBpmnMarkerKind.EXPAND]));
 
-const addContainerElements = (bpmnModel: BpmnModel, containerWithLanes: ContainerWithLanes & BaseElement): void => {
+const addContainerElements = (bpmnModel: BpmnModel, containerWithLanes: ContainerWithLanes & BaseElement, parentId: string): void => {
```

Comment:
Because `containerWithLanes` could be a `subprocess`, `call activity`, `lane`, `pool/participant` (who is the `parent` of its elements), but it also could be a `process` (who is NOT the `parent` of its elements).

Reply:
reverted, will be introduced in #2139